README.md

```diff
@@ -1,90 +1,71 @@
 # Global Code Queens Open Source Project
 
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Challenge](#challenge)
+- [Design](#design)
+- [Features](#features)
+- [Installation](#installation)
+- [Contributing](#contributing)
+
+## Introduction
+
 This is an open source project developed as part of the [Open Source Project](https://www.linkedin.com/posts/frontendqueens_opensource-collaboration-community-activity-7201980886232317952-VBld?utm_source=share&utm_medium=member_desktop) initiative by the [FrontEnd Queens](https://www.linkedin.com/company/frontendqueens/)
 
-## The Challenge
+## Challenge
 
 The challenge is to build out a multi-page website and get it looking as close to the design as possible.
 
 ## Design
 
 The challenge is part of [Frontend Mentor](https://www.frontendmentor.io/) premium version, hence we are not adding a link of the Design file here.
 
-## More details will be added in due course of time
-
-# Getting Started
-
-1. Clone the repo. From the terminal, run:
```

Comment:
Did you consider moving the detailed information on git/GitHub to the bottom of the README (below the section on Contributing), or into a separate markdown file? There may be parts of that info that remain very helpful, particularly the information on the expectations for Pull Requests.

Reply:
Yes, you are right. Its very important to keep it, I am just working on the separate markdown file, which I will attach in the prerequisites

Reply:
its done now